.github/workflows/main.yml

```diff
@@ -188,6 +188,21 @@ jobs:
       working-directory: ./fuzz
     - run: cargo fuzz build --dev
 
+  rebuild_isle:
+    name: Rebuild ISLE
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+    - name: Rebuild ISLE DSL files
+      run: cargo build -p cranelift-codegen --features "rebuild-isle"
+    - name: Reformat
+      run: cargo fmt -p cranelift-codegen
```

Comment:
Is this step still necesary now that rustfmt is baked in?

Reply:
Yes, since the `rustfmt`ing is happening on a best-effort basis, and if `rustfmt` isn't on `$PATH` it doesn't break the build.